ui/app/templates/components/raft-join.hbs

```diff
@@ -30,28 +30,29 @@
         <br />
         How do you want to get started?
       </h2>
-      <RadioButton
-        @value="join"
-        @groupValue={{this.preference}}
-        @changed={{action (mut this.preference)}}
-        @name="setup-pref"
-        @radioId="join"
-        @classNames="vlt-radio is-block"
-      >
+      <div class="vlt-radio is-block">
+        <RadioButton
+          id="join"
+          name="setup-pref"
+          class=""
```

Comment:
does `class` need to be here if it's empty? I noticed it wasn't in the component below